detox/test/e2e/c-actions.js

```diff
@@ -17,13 +17,18 @@ describe('Actions', () => {
     await expect(element(by.text('Long Press Working!!!'))).toBeVisible();
   });
 
+  it(':ios: should long press with duration on an element', async () => {
```

Comment:
I don't want to create more discrepancy between iOS and Android API, I'm willing to help you add Android support as well if you need help.

Reply:
Yeah I also really want this feature can be supported for Android, if you could help will be great. Thanks. :)